numpy/add_newdocs.py

```diff
@@ -374,13 +377,50 @@ def luf(lamdaexpr, *args, **kwargs):
             while not it.finished:
                 it[0] = lamdaexpr(*it[1:])
                 it.iternext()
+            it.close()
             return it.operands[0]
 
         >>> a = np.arange(5)
         >>> b = np.ones(5)
         >>> luf(lambda i,j:i*i + j/2, a, b)
         array([  0.5,   1.5,   4.5,   9.5,  16.5])
 
+    If operand flags `"writeonly"` or `"readwrite"` are used the operands may
+    be views into the original data with the WRITEBACKIFCOPY flag. In this case
+    it is best practice to use the nditer as a context manager. The temporary
```

Comment:
Maybe change "it is best practice to use" to "should be used" or "must be used"

Reply:
fixed